lib/client.js

```diff
@@ -357,22 +391,42 @@ extend(Raven.prototype, {
       res.sentry = eventId;
       return next(err);
     };
+  },
+
+  captureBreadcrumb: function (breadcrumb) {
+    if (!this.installed) return;
+
+    breadcrumb = extend({
+      timestamp: +new Date / 1000
+    }, breadcrumb);
+
+    var currCtx = this.getContext();
+    if (!currCtx.breadcrumbs) currCtx.breadcrumbs = [];
+    currCtx.breadcrumbs.push(breadcrumb);
+    if (currCtx.breadcrumbs.length > this.maxBreadcrumbs) {
```

Comment:
So `maxBreadcrumbs` is effectively per context? Is there a concern that someone could have a *lot* of contexts, and as a result breadcrumbs end up consuming a *lot* of memory?